vcs/repository.go

```diff
@@ -36,7 +36,7 @@ type Repository interface {
 	// ErrCommitNotFound if no such commit exists.
 	GetCommit(CommitID) (*Commit, error)
 
-	// Commits returns all commits matching the options, as well as
+	// Commits returns all commits matching the options, and optionally
```

Comment:
This comment is technically incorrect, as the default computes the total. Should read "Optionally, the caller can request the total not to be computed, as this can be expensive."